n3fit/src/n3fit/backends/keras_backend/operations.py

```diff
@@ -126,26 +124,17 @@ def numpy_to_input(
     Parameters
     ----------
         numpy_array: np.ndarray
-        no_reshape: bool
-            if true, don't add batch dimension, take the first dimension of the array as the batch
         name: bool
             name to give to the layer
-        custom_shape: tuple
-            To specify a more general shape with None values
     """
-    if no_reshape:
-        batched_array = numpy_array
-        batch_size = numpy_array.shape[0]
-        shape = numpy_array.shape[1:]
-    else:
-        batched_array = np.expand_dims(numpy_array, 0)
-        batch_size = 1
-        shape = numpy_array.shape
-    if custom_shape is not None:
-        shape = custom_shape
+    batched_array = np.expand_dims(numpy_array, 0)
+    batch_size = 1
```

Comment:
Since `batch_size` doesn't depend on any condition you may as well skip defining it and just pass `batch_size=1` to Input`. 

P.S. the docstring of this function can be changed from "By default it adds..." to "It adds ..."